package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internetarchive/bookreader",
-  "version": "4.20.1",
+  "version": "4.20.2",
```

Comment:
hi @nsharma123 - let's not hard code the version for BookReader Repo as we have a formal release process.

If you can, please rsync the build directory to your review app for testing and QA. thanks!

Reply:
Its just come in this commit, I was not indent to coming this here. Will remove